website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -50,133 +66,185 @@ multigroup: true
+The next step will be port our code to the new project. 
 
-### Migrate API type definitions
+### Migrating API's
 
-Copy over the API spec and status from the SDK project's `pkg/apis/<group>/<version>/<kind>_types.go` to Kubebuilder's `api/<version>/<kind>_types.go`.
+Now, let’s copy the API definition from `pkg/apis/<group>/<version>/<kind>_types.go` to `api/<version>/<kind>_types.go`. Note that is only required to copy the code from the Spec and Status fields.
 
-In our example, `MemcachedSpec` and `MemcachedStatus` from [`pkg/apis/cache/v1aplha1/memcached_types.go`][memcached_types] are copied to `api/v1alpha1/memcached_types.go`.
+**What to do about the Markers**
+
+- The `+k8s:deepcopy-gen:interfaces=...` marker was replaced with `+kubebuilder:object:root=true`.
+- We don’t need `// +k8s:openapi-gen=true` marker any more since they’re not used anymore
+
+The Memcached API types will look like:
 
 ```go
+// MemcachedSpec defines the desired state of Memcached
 type MemcachedSpec struct {
-	// Specify size, the number of pods
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Size is the size of the memcached deployment
 	Size int32 `json:"size"`
 }
 
+// MemcachedStatus defines the observed state of Memcached
 type MemcachedStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
 	// Nodes are the names of the memcached pods
+	// +listType=set
 	Nodes []string `json:"nodes"`
 }
+
+// +kubebuilder:object:root=true
+
+// Memcached is the Schema for the memcacheds API
+type Memcached struct {...}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+
+// MemcachedList contains a list of Memcached
+type MemcachedList struct {...}
 ```
-**Note**:
-If there are any libraries or pkgs present in `pkg/apis/cache/v1alpha1`, copy them over to `api/v1alpha1`.
 
-### CRD markers
+### Migrating the Controllers
 
-Include any of the CRD generation or validation markers defined for your API type and fields in `api/<version>/<kind>_types.go`.
+In our example, we will migrate the code from `pkg/controller/memcached/memcached_controller.go` to `controllers/memcached_controller.go`. 
 
-For `memcached-operator` the following markers present in `pkg/apis/cache/v1aplha1/memcached_types.go` are copied to `api/v1alpha1/memcached_types.go`.
+Now, the struct Reconcile<Kind> is called <KindReconciler>, so if you have customizations copy the fields from the for example `ReconcileMemcached` struct to `MemcachedReconciler`.
 
-Example:
-```Go
-// +kubebuilder:subresource:status
-// +kubebuilder:resource:path=memcacheds,scope=Namespaced
+We’ll need replace the `// your logic here` in the new layout for our reconcile logic. See that in our example the:
```

Comment:
```suggestion
We’ll need to replace the `// your logic here` in the new layout for our reconcile logic. See that in our example the:
```

I'm not sure there should be "We" or "you" to be used. But let's be consistent in referring the reader